Depend-on/Bash.sh

```diff
@@ -0,0 +1,221 @@
+                            sudo dnf install -y libavcodec-devel libavformat-devel libavutil-devel libswscale-devel
+                            sudo dnf install -y freerdp2-devel
+                            sudo dnf install -y pango-devel
+                            sudo dnf install -y libssh2-devel
+                            sudo dnf install -y libtelnet-devel
+                            sudo dnf install -y libvncserver-devel
+                            sudo dnf install -y libwebsockets-devel
+                            sudo dnf install -y pulseaudio-libs-devel
+                            sudo dnf install -y openssl-devel
+                            sudo dnf install -y libvorbis-devel
+                            sudo dnf install -y libwebp-devel
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                elif command -v yum &>/dev/null; then
+                    echo "The following packages will be installed:"
+                    echo "FFmpeg"
+                    echo "libavcodec-devel libavformat-devel libavutil-devel libswscale-devel"
+                    echo "freerdp2-devel"
+                    echo "pango-devel"
+                    echo "libssh2-devel"
+                    echo "libtelnet-devel"
+                    echo "libvncserver-devel"
+                    echo "libwebsockets-devel"
+                    echo "pulseaudio-libs-devel"
+                    echo "openssl-devel"
+                    echo "libvorbis-devel"
+                    echo "libwebp-devel"
+
+                    read -p "Do you understand? (yes/no): " choice
+                    case "$choice" in 
+                        yes|YES|y|Y)
+                            sudo yum install -y FFmpeg
+                            sudo yum install -y libavcodec-devel libavformat-devel libavutil-devel libswscale-devel
+                            sudo yum install -y freerdp2-devel
+                            sudo yum install -y pango-devel
+                            sudo yum install -y libssh2-devel
+                            sudo yum install -y libtelnet-devel
+                            sudo yum install -y libvncserver-devel
+                            sudo yum install -y libwebsockets-devel
+                            sudo yum install -y pulseaudio-libs-devel
+                            sudo yum install -y openssl-devel
+                            sudo yum install -y libvorbis-devel
+                            sudo yum install -y libwebp-devel
+                            ;;
+                        *)
+                            echo "Installation cancelled."
+                            ;;
+                    esac
+                else
+                    echo "Neither dnf nor yum found. Unsupported package manager."
+                fi
+                ;;
+            Ubuntu|Debian)
```

Comment:
Xubuntu, Lubuntu. I'm not as familiar with Debian-based distros, but there may be others.

Reply:
thats the Debian tag that should pick it up but if not I might add a redundnt ask so for example if its not found you type in the package manager and thats what im thinking